src/django_components/component.py

```diff
@@ -318,6 +346,7 @@ def render(
                 # See https://github.com/EmilStenstrom/django-components/issues/280#issuecomment-2081180940
                 "component_vars": {
                     "is_filled": slot_bools,
+                    "attrs": attrs,
```

Comment:
I exposed the parsed attributes via `{{ component_vars.attrs }}`.

TODO: This `attrs` value should be a copy of the original, so that if user makes changes to the `attrs` passed to `get_context_data()`, then this will still remain unchanged.   ...Or better yet, make `attrs` dict immutable, so it can't be changed in neither `get_context_data` nor the template.